TF_Settings_Web/src/Styles/_variables.scss

```diff
@@ -6,6 +6,7 @@ $ultraleap-green: #00eb85;
 $light-grey-gradient: linear-gradient(180deg, #5c5c5c 0%, #454545 100%);
 $dark-grey-gradient: linear-gradient(180deg, #373737 0%, #2c2c2c 100%);
 $base-font: MontserratRegular, Arial;
+$button-border-radius: 10px;
```

Comment:
This isn't used anywhere (expect the class above)

Reply:
Removed